live-examples/css-examples/fragmentation/break-before.html

```diff
@@ -0,0 +1,36 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="break-before">
+    <div class="example-choice">
+        <pre><code class="language-css">break-before: auto;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">break-before: page;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">break-before: left;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+</section>
+
+<div id="output" class="output large hidden">
+    <section id="default-example">
+        <div>
+            <p>The effect of this property can be noticed when the document is being printed or a preview of a print is displayed.</p>
+            <button id="print-btn">Show Print Preview</button>
+            <div class="box-container">
+                <div class="box">Content Before</div>
+                <div class="box" id="example-element">Content with break-before</div>
+                <div class="box">Content After</div>
```

Comment:
How about rephrasing to the following to add more clarity:

```suggestion
                <div class="box">Content before the property</div>
                <div class="box" id="example-element">Content with 'break-before'</div>
                <div class="box">Content after the property</div>
```

This suggestion can be applied to other html files in this pull request

Reply:
That's better. I have applied your suggestion to all examples. Thanks